IdentityCore/tests/util/MSIDKeychainTokenCache+MSIDTestsUtil.m

```diff
@@ -0,0 +1,34 @@
+//
+//  MSIDTestUtil.m
+//  IdentityCore
+//
+//  Created by Sergey Demchenko on 12/4/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "MSIDKeychainTokenCache+MSIDTestsUtil.h"
+
+@implementation MSIDKeychainTokenCache (MSIDTestUtil)
+
++ (void)reset
+{
+    [self deleteAllKeysForSecClass:kSecClassGenericPassword];
+    [self deleteAllKeysForSecClass:kSecClassInternetPassword];
```

Comment:
are we using anything but generic password?

Reply:
removed deletion for kSecClassInternetPassword.